ostd/src/mm/vm_space.rs

```diff
@@ -52,6 +55,14 @@ use crate::{
 pub struct VmSpace {
     pt: PageTable<UserMode>,
     page_fault_handler: Once<fn(&VmSpace, &CpuExceptionInfo) -> core::result::Result<(), ()>>,
+    activated_cpus: Arc<SpinLock<CpuSet>>,
```

Comment:
Add a todo.

```rust
    // TODO: For efficiency, implement an `AtomicCpuSet` type to replace `SpinLock<CpuSet>`
    activated_cpus: Arc<SpinLock<CpuSet>>,
```

Reply:
There exists 256-core machines and I suspect that 256-bit atomic instructions are really rare. A TODO is good, but maybe only suitable for normal cases. We might feature gate it.